doc/style-guide.md

```diff
@@ -333,6 +333,16 @@ line of code, indented by two spaces.
   ...                  | false = filter p xs
   ```
 
+* Instance arguments, and their types, should use the vanilla ASCII/UTF-8 `{{_}}`
```

Comment:
Are the unrelated changes meant to be included here?

Reply:
Yes, at least AFAIAC, because it's miscellaneous documentation which has come up in the course of other PRs. Messy perhaps, and one reason I've been more considered (new label; new issue) about flagging such things now...